docs/slurk_bots.rst

```diff
@@ -236,3 +236,16 @@ can listen to a command with ``on_command``:
 - ``user``: dictionary of ``id`` and ``name`` of the user, who submitted the command
 - ``room``: the id of the room, where the command was entered, or ``None``
 - ``timestamp``
+
+
+Testing bots
+~~~~~~~~~~~~
+
+You can also test your custom bots locally using ``docker``. First, you should create a ``Dockerfile``
+inside the folder where your bots reside. The content of the ``Dockerfile`` should follow the one available
+under ``clp-research/slurk-bots``. Afterwards, you can build the docker container using 
```

Comment:
let's add the exact name of the file

Reply:
I already updated it with the exact filename